### ISSUE TITLE
Incompatible with Flutter SVG

### ISSUE THREAD

**Issue by garv-shah** [simpleclub/math_keyboard#35]
## Bug description

The package seems to be incompatible with Flutter SVG, as the dependency for petitparser is out of date, set to ^4.0.2

### To Reproduce

Steps to reproduce the behaviour:
1. Install math_keyboard
2. Install flutter_svg
3. Try to run app and observe error

## Expected behavior

For both packages to work together

### Error Log

```
Because math_keyboard >=0.0.1 depends on petitparser ^4.0.2 and xml >=5.4.0 depends on petitparser ^5.0.0, math_keyboard >=0.0.1 is incompatible with xml >=5.4.0.
And because flutter_svg 1.1.1+1 depends on xml ^6.0.1 and no versions of flutter_svg match >1.1.1+1 <2.0.0, math_keyboard >=0.0.1 is incompatible with flutter_svg ^1.1.1+1.
So, because app_name depends on both flutter_svg ^1.1.1+1 and math_keyboard ^0.1.6, version solving failed.
pub get failed (1; So, because app_name depends on both flutter_svg ^1.1.1+1 and math_keyboard ^0.1.6, version solving failed.)
```

**Comment by garv-shah** [comment]
https://github.com/simpleclub/math_keyboard/pull/36
PR for a fix! :D

**Comment by edhom** [comment]
Fixed in `0.1.7`🙂 Thanks @garv-shah!

**Comment by garv-shah** [comment]
ty :D
